Utilities/VisualStudioProjectTemplates/EdFi.ProjectTemplates.Extensions/Template_Project.csproj

```
@@ -56,6 +55,7 @@
       <DesignTime>True</DesignTime>
       <SubType>Designer</SubType>
     </EmbeddedResource>
+	<EmbeddedResource Include="assemblyMetadata.json" />
```

Let's keep the indentation for files consistent
```suggestion
    <EmbeddedResource Include="assemblyMetadata.json" />
```